src/app/hooks/useAuth.tsx

```
@@ -0,0 +1,114 @@
+        },
+        body: JSON.stringify({ email }),
+      });
+
+      if (!response.ok) {
+        const errorData = await response.json();
+        throw new Error(errorData.message || "Email verification failed");
+      }
+
+      return await response.json();
+    },
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)